package.json

```diff
@@ -20,11 +20,11 @@
   },
   "lint-staged": {
     "*.{ts,tsx}": [
-      "eslint --fix --max-warnings 0",
+      "eslint --fix",
       "git add"
     ],
     "*.{js,jsx}": [
-      "eslint --fix --max-warnings 0",
+      "eslint --fix",
```

Comment:
Please do not remove these.

Reply:
Ok..I will readd but we need to follow up on this https://github.com/OpenSRP/web/issues/87